Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityController.cs

```diff
@@ -249,7 +262,7 @@ private void UpdateGripData(InteractionSourceState interactionSourceState, Mixed
                         interactionSourceState.sourcePose.TryGetPosition(out currentGripPosition, InteractionSourceNode.Grip);
                         interactionSourceState.sourcePose.TryGetRotation(out currentGripRotation, InteractionSourceNode.Grip);
 
-                        if (MixedRealityToolkit.Instance.MixedRealityPlayspace != null)
+                        if ((MixedRealityToolkit.Instance?.MixedRealityPlayspace ?? null) != null)
```

Comment:
This bypasses the lifetimes checks of the monobehaviours

https://github.com/JetBrains/resharper-unity/wiki/Possible-unintended-bypass-of-lifetime-check-of-underlying-Unity-engine-object

Reply:
This should now be addressed